keyboards/handwired/yamsek/rp2040/config.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Johannes Schneider (@JohSchneider)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define RGB_DI_PIN GP16
+//PICO_DEFAULT_WS2812_PIN
+#define RGBLED_NUM 1
+
+#define I2C1_SDA_PIN GP2
+#define I2C1_SCL_PIN GP3
+
+#define I2C1_CLOCK_SPEED 100000
+
+#define I2C1_SCL_PAL_MODE (PAL_MODE_ALTERNATE_I2C | PAL_RP_PAD_PUE | PAL_RP_PAD_SLEWFAST)
+#define I2C1_SDA_PAL_MODE (PAL_MODE_ALTERNATE_I2C | PAL_RP_PAD_PUE | PAL_RP_PAD_SLEWFAST)
+// removing 'PAL_RP_PAD_DRIVE4' which the portexpander in the left keyboard half doesn't like
+// alternative solution would be to add a 100pF ceramic-capacitor between SCL and GND
```

Comment:
The RP2040 datasheets says this:
![image](https://user-images.githubusercontent.com/616082/233829898-3d2ba07a-9a91-4665-aaef-7503923d93b2.png)
I suppose that “slew rate limited” means that `PAL_RP_PAD_SLEWFAST` must not be used.  Maybe that's the proper fix here?

As for `PAL_RP_PAD_DRIVE4`, there was some side discussion in https://github.com/qmk/qmk_firmware/pull/20106#discussion_r1161317283 — the I2C spec may require even higher driving current in some cases, but as you are already limiting the speed to 100 kHz, your configuration may actually be fine with the lower driving current.

Reply:
thanks for the linked discussion - so i'd have to check my build, and see if the long-ish cable or connectors in-between are the rootcause, since i'm running i2c over a TRRS cable from one side to the other...

i'll drop the defines in the meantime

Reply:
You may need to have these defines for now, just with different values — although #20314 made it possible to set custom values for these defines, it did not change the default values (they still have the potentially problematic `PAL_RP_PAD_SLEWFAST`).